content/cli/ssh.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: bt_wiki
+title: ssh
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 40
+---
+
+The `cfy ssh` command is used to connect to a Cloudify manager via ssh.
```

Comment:
I suggest that other than the actual 'cfy ssh' command any other mention of ssh in the documentation should be capitalized. (as a reference you can have a look at Vagrant's documenation)